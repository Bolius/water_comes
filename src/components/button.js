import styled from 'styled-components';
import { Button as BootStrapButton } from 'reactstrap'

export const Button = styled(BootStrapButton).attrs(() => ({
    block: true,
  }))`
    background-color: #0E6BA3 !important;
    text-transform: uppercase;

    :hover {
      background-color: #24A2B7 !important;
    }
  `;
