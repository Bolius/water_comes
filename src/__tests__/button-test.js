import React from "react";
import { create } from "react-test-renderer";
import ReactDOM from "react-dom";
import { Button } from "../components/button.js";

test("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Button />, div);
});

describe("Button component", () => {
  test("it matches the snapshot", () => {
    const component = create(<Button />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
