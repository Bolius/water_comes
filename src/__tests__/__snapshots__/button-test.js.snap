// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component it matches the snapshot 1`] = `
<button
  aria-label={null}
  className="sc-bdVaJa ksyBqg btn btn-secondary btn-block"
  onClick={[Function]}
/>
`;
